### ISSUE TITLE
Split big paragraph in Screen Summary into 2 paragraphs

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#242]
@jessegreenberg, could you split the big paragraph into two, so that

> In the Play Area, you find an observation window that contains a light source and a molecule. You can adjust the rate of photons being emitted at the molecule, choose a different molecule, or change light source. From the Control Area you can use buttons to pause/play and step forward what’s happening in the observation window. You can also access details about the light spectrum and reset the sim.

Is split like this:
> In the Play Area, you find an observation window that contains a light source and a molecule. You can adjust the rate of photons being emitted at the molecule, choose a different molecule, or change light source. 

>From the Control Area you can use buttons to pause/play and step forward what’s happening in the observation window. You can also access details about the light spectrum and reset the sim.

Some sims are naturally a bit shorter, but I always try to split the Play Area info and Control Area info. More than once, screen reader users have told me that the screen summary info should be chunked into manageable pieces, and this is how I have decided to do it.

**Comment by jessegreenberg** [comment]
Sounds good, Ill split it up.

**Comment by jessegreenberg** [comment]
Done in the above commit.
